modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/ConnectStep.spec.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import {render} from '@testing-library/react';
+import React from 'react';
+
+import ConnectStep from './ConnectStep';
+
+describe('Connect Step', () => {
+	it('render ConnectStep without crashing', () => {
+		const {container, getByText} = render(
+			<ConnectStep onCancel={() => {}} onChangeStep={() => {}} />
+		);
+
+		const connectStepDescription = getByText(
+			'use-the-token-genereted-in-your-analytics-cloud-to-connect-this-workspace'
+		);
+
+		// const connectSheet = container.getElementsByClassName('sheet');
```

Comment:
Could remove this line?

Reply:
Thank you, @interaminense! I actually forgot to remove this line. Already done!